client/battle/CCreatureAnimation.cpp

```diff
@@ -425,28 +427,22 @@ inline void CCreatureAnimation::putPixel(ui8 * dest, const SDL_Color & color, si
 
 bool CCreatureAnimation::isDead() const
 {
-	return getType() == CCreatureAnim::DEAD
-	    || getType() == CCreatureAnim::DEATH;
+	return getType() == CCreatureAnim::DEAD || getType() == CCreatureAnim::DEATH;
 }
 
 bool CCreatureAnimation::isIdle() const
 {
-	return getType() == CCreatureAnim::HOLDING
-	    || getType() == CCreatureAnim::MOUSEON;
+	return getType() == CCreatureAnim::HOLDING || getType() == CCreatureAnim::MOUSEON;
 }
 
 bool CCreatureAnimation::isMoving() const
 {
-	return getType() == CCreatureAnim::MOVE_START
-	    || getType() == CCreatureAnim::MOVING
-	    || getType() == CCreatureAnim::MOVE_END;
+	return getType() == CCreatureAnim::MOVE_START || getType() == CCreatureAnim::MOVING || getType() == CCreatureAnim::MOVE_END;

```

Comment:
Why this was merged into a single line?
Not sure how common large ` A || B || C && D` statements in code but having each element on a separate line usually looks better.